src/lib/discord/commando/events/messageCreate.js

```diff
@@ -53,9 +53,9 @@ module.exports = async (client, msg) => {
 
 				// Convert to lower case, translate back to english, and substitute _ for spaces
 
-				args = args.map((arg) => client.translatorFactory.reverseTranslateCommand(arg.toLowerCase()
-					.replace(/_/g, ' '), true)
-					.toLowerCase())
+        args = args.map((arg) => client.translatorFactory.reverseTranslateCommand(arg.toLowerCase()
```

Comment:
Since this is replaced, feels like `npm run lint` is waiting to be run to get the indentation right

Reply:
I have less than a clue about linting, I thought my editor was handling spacing for me but I can run that, once my server is back online, been off due to a move and power issues, and have a long to-do list when it comes back (likely tonight). If it can be fixed during merge that's also fine, the base branch and related folder will be deleted once accepted anyways.

Reply:
The merge does not apply the lint requirements. It's best you do this and push because it might complain about something (and not just fix the indentation etc) which you will need to resolve.

Reply:
once this is complete then I will merge